spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -216,7 +230,7 @@ public void listen2(@Payload String foo, @Header(KafkaHeaders.PARTITION_ID) int
 			this.latch2.countDown();
 		}
 
-		@KafkaListener(id = "baz", topicPartitions = @TopicPartition(topic = "annotated3", partition = "0"))
+		@KafkaListener(id = "baz", topicPartitions = @TopicPartition(topic = "annotated3", partitions = "0"))
```

Comment:
Yeah - also a bug in the test - waiting for latch 4 not 5; will fix - I want to make a couple more polishes too...